eng/pipelines/coreclr/ci.yml

```diff
@@ -97,6 +97,16 @@ jobs:
     jobTemplate: /eng/pipelines/common/templates/runtimes/build-test-job.yml
     buildConfig: checked
     platforms:
+    - OSX_x64
+    testGroup: outerloop
+    jobParameters:
+      liveLibrariesBuildConfig: Release
+      testBuildPhase: targetGeneric
+- template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
And in CI we are building the target specific tests here.

Reply:
OK, I see. Let's see how this ends up changing the pipeline running time as @jashook mentioned multiple times in the past that too fine-grained job splitting has a potential of aggravating per-job infra costs. My 1 cent would be that running build-test to build the targetSpecific tests as part of run-test-job should be almost for free and it would also spare you the need to modify the individual pipelines, the phasing change would be basically encapsulated in the build-test-job vs. run-test-job interplay.